packages/rum/src/boot/rum.ts

```diff
@@ -198,6 +206,36 @@ function trackErrors(lifeCycle: LifeCycle, handler: (startTime: number, event: R
   })
 }
 
+function trackManualErrors(
+  lifeCycle: LifeCycle,
+  handler: (startTime: number, event: RumErrorEvent, savedGlobalContext?: Context, customerContext?: Context) => void
+) {
+  lifeCycle.subscribe(
+    LifeCycleEventType.MANUAL_ERROR_COLLECTED,
+    ({ error: { error, startTime, context: customerContext, source }, context: savedGlobalContext }) => {
+      const stackTrace = error instanceof Error ? computeStackTrace(error) : undefined
+      const { message, stack, kind } = formatUnknownError(stackTrace, error, 'Captured')
```

Comment:
I used 'Captured' here, but I'm not convinced about it. I don't really like 'Added', but I'm open to alternatives.

Reply:
`Manual`, `Custom`?

Reply:
`Manual { foo: bar }` seems weird. Those errors may not be `Custom` and are not `RUM Error`... Maybe we could use `Caught { foo: bar }` (in contrast to `Uncaught { foo: bar }`)? Or  we could be a bit more verbose:`Manually caught ${errorSource} error { foo: bar }`

Reply:
wdyt of `Provided { foo: bar }`?